src/lib.rs

```diff
@@ -231,7 +233,14 @@ impl Adapter<HttpConnector, Body> {
     }
 
     async fn is_web_ready(&self, url: &Url, protocol: &Protocol) -> bool {
+        let start = Instant::now();
+        let interval = 5; // TODO: make this configurable?
+        let mut next_checkpoint = interval;
         Retry::spawn(FixedInterval::from_millis(10), || {
+            if start.elapsed().as_secs() > next_checkpoint {
+                tracing::debug!("app is not ready after {next_checkpoint}s");
```

Comment:
Use info here. And also include the URL in the log, so that people know which URL is being hit.

Reply:
The purpose is to give visibility of readiness check status to the users. Right now, if the is an issue with readiness check, the function times out without any warning. This log should give new users a good hint of the issue.

Reply:
Updated.